pennylane/tape/qnode.py

```diff
@@ -560,7 +562,7 @@ def to_autograd(self):
         if self.qtape is not None:
             AutogradInterface.apply(self.qtape)
 
-    INTERFACE_MAP = {"autograd": to_autograd, "torch": to_torch, "tf": to_tf}
+    INTERFACE_MAP = {"autograd": to_autograd, "torch": to_torch, "tf": to_tf, "jax": lambda x: x}

```

Comment:
This is interesting. How come we don't need a specialized interface construction function for jax?

Reply:
Not sure, this was recommended to me by @josh146

Reply:
Having said that, we should probably update the jax 'interface' to allow for some additional validation, since the Jax device does not support non-backprop differentiation at the moment:

```python
def to_jax(self):
    if self.diff_method != "backprop":
        raise qml.QuantumFunctionError(
            "The JAX interface can only be used with "
            "diff_method='backprop' on supported devices"
        )
    self.interface = "jax"

INTERFACE_MAP = {"autograd": to_autograd, "torch": to_torch, "tf": to_tf, "jax": to_jax}
```

(this avoids having `interface="jax"` and `diff_method="parameter-shift"`, which currently doesn't raise any error).

Reply:
Added.